### PR TITLE
phone input and validate

### PR DESCRIPTION
**Pull request by ChaiyapatOam**

### CODE REVIEW

**Comment by riflowth** [comment]
Please use the information title for the PR ticket

**Comment by riflowth** [comment]
Please use the information title for the PR ticket @ChaiyapatOam

File: apps/registration/src/components/PhoneInput.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  name: string;
+  id: string;
+  placeholder: string;
+  obj: string | number;
+  setObj: any;
+  required?: boolean;
+}) => {
+  const [isValid, setIsValid] = useState<boolean>(true);
+
+  const handleChange = (e) => {
```
Comment (wit03) [inline apps/registration/src/components/PhoneInput.tsx:19]: There's an error with handleChange function see #21 P'Ohm's comment as a reference.

File: apps/registration/src/components/PhoneInput.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <div className="mt-1">
+        <input
+          type="text"
+          name={name}
+          id={id}
+          placeholder={placeholder}
+          value={obj}
+          onChange={handleChange}
+          className="rounded-md appearance-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 focus:outline-none focus:ring-purple-500 focus:border-purple-500 focus:z-10 sm:text-sm"
+          required={required}
```
Comment (wit03) [inline apps/registration/src/components/PhoneInput.tsx:44]: No needs to use required <input> attribute, see [Inputbox Component](https://github.com/cpe-kmutt-student/bangmod-hackathon-2023/blob/main/apps/registration/src/components/Inputbox.tsx) as a reference on how to handle required props.

**Review by wit03 — CHANGES_REQUESTED** [review]

File: apps/registration/src/components/PhoneInput.tsx (hunk 2 above)
Comment (ChaiyapatOam) [reply apps/registration/src/components/PhoneInput.tsx:44]: Oh, Okayy!

**Review by wit03 — APPROVED** [review]

**Comment by wit03** [comment]
LGTM yayyy

**Pull request merged**